File: Front End/memorizewords/src/app/services/http/model/back/word-response.ts
import { WordAnswer } from "../dto/word-answer";

export interface WordResponse {
    wordId: number;
    word: string;
    meaning: string;
    percentage:string;
    writingInLanguage:string;
    isLearned:boolean;
    askWordAgain:boolean;
    wordAnswers: WordAnswer[];
}

File: Back End/MemorizeWords/MemorizeWords/Presentation/Models/Response/WordResponse.cs
using MemorizeWords.Presentation.Models.Dto;

namespace MemorizeWords.Presentation.Models.Response
{
    public class WordResponse
    {
        public int WordId { get; set; }
        public string Word { get; set; }
        public string Meaning { get; set; }
        public string WritingInLanguage { get; set; }
        public string Percentage { get; set; }
        public List<WordAnswerDto> WordAnswers { get; set; }
    }
}
